packages/website/scripts/generateModulePages.mjs

```diff
@@ -6,100 +6,63 @@ import {
-description: Dates are a simple date time type.
----
-import { SquiggleEditor, FnDocumentationFromName } from "@quri/squiggle-components";
+function toMarkdown(documentation) {
+  const fullName = documentation.nameSpace + "." + documentation.name;
+  return `### ${documentation.name}
+${escapedStr(documentation.description || "")}
+<FnDocumentationFromName functionName="${fullName}" showNameAndDescription={false} size="small" />
+`;
+}
 
-# Date`,
-  },
-  {
-    name: "Duration",
-    intro: `---
-description: Durations are a simple time type, representing a length of time. They are internally stored as milliseconds, but often shown and written using seconds, minutes, hours, days, etc.
----
-import { SquiggleEditor, FnDocumentationFromName } from "@quri/squiggle-components";
+const generateModulePage = async ({ name, description, intro, sections }) => {
+  const namespaceNames = getAllFunctionNamesWithNamespace(name);
+  let fnDocumentationItems = namespaceNames
+    .map(getFunctionDocumentation)
+    .filter(({ isUnit }) => !isUnit);
 
-# Duration
-Durations are a simple time type, representing a length of time. They are internally stored as milliseconds, but often shown and written using seconds, minutes, hours, days, etc.
+  const processSection = (section) => {
+    const sectionFnDocumentationItems = fnDocumentationItems.filter(
+      ({ displaySection }) => displaySection === section.name
+    );
+    if (sectionFnDocumentationItems.length === 0) {
+      throw new Error(
+        `Error: No functions in section: ${name} ${section.name}. You likely made an error in the section name.`
+      );
+    }
 
+    const sectionHeader = section.name && `## ${section.name}\n\n`;
+    const sectionDescription =
+      section.description && `${section.description}\n\n`;
+    const sectionItems = sectionFnDocumentationItems.map(toMarkdown).join("\n");
+    return `${sectionHeader || ""}${sectionDescription || ""}${sectionItems}`;
+  };
 
-| **Unit Name** | **Example** | **Convert Number to Duration** | **Convert Duration to Number** |
-|---------------|----------------------------|--------------------------------------------|--------------------------------------------|
-| Minute        | \`5minutes\`                   | \`fromMinutes(number)\`                      | \`toMinutes(duration)\`                      |
-| Hour          | \`5hour\`                     | \`romHours(number)\`                        | \`toHours(duration)\`                        |
-| Day           | \`5days\`                      | \`fromDays(number)\`                         | \`toDays(duration)\`                         |
-| Year          | \`5years\`                     | \`fromYears(number)\`                        | \`toYears(duration)\`                        |
+  let functionSection;
+  if (sections?.length > 0) {
+    functionSection = sections.map(processSection).join("\n\n");
+  } else {
+    functionSection = fnDocumentationItems.map(toMarkdown).join("\n\n");
```

Comment:
Can we have an assertion here somewhere that there are no functions that don't belong to sections defined in the module config?

And then either fail early with an exception, or send those to the "Other" section at the end.

`displaySection` is a string, so it's easy to make a typo in `fr/*.ts` and not notice for a while that documentation is missing from the website.

Reply:
Will see about doing in a future PR